### PR TITLE
flink_execute

### PR DESCRIPTION
**Pull request by niladrem**
##### PR Checklist
- [x] Tests added
- [x] [Changelog](CHANGELOG.md) updated

### CODE REVIEW

File: tests/streamingcli/jupyter/test_notebook_converter.py [hunk 1]
```diff
@@ -327,6 +330,30 @@ def filter_print(condition, message):
...
+    def test_notebook_with_flink_code(self):
+        # given
+        file_path = "tests/streamingcli/resources/jupyter/notebook7.ipynb"
+        # expect
+        converted_notebook = convert_notebook(file_path)
+        assert (
+            converted_notebook.content
+            == """from pyflink.common.typeinfo import Types
+from pyflink.datastream import StreamExecutionEnvironment
+from pyflink.table import StreamTableEnvironment, DataTypes
```
Comment (andrzejdackiewicz) [inline tests/streamingcli/jupyter/test_notebook_converter.py:346]: I see that DataTypes and udf are used in one of the tests and they are imported in the flink.app.template.
What about removing those imports from all tests that do not refer to udf and DataTypes and adding them only where we use udf and DataTypes? Would that be a good idea? Besides that LGTM!

Comment (niladrem) [reply tests/streamingcli/jupyter/test_notebook_converter.py:346]: Hmmm, I think we don't need these imports in the template at all. We should have them in the code.

Comment (MarekMaj) [reply tests/streamingcli/jupyter/test_notebook_converter.py]: User who defines function via jupyter should add imports as those are part of function definition. Imo no need to have them in template, especially in scenario when only sql is submitted

Comment (niladrem) [reply tests/streamingcli/jupyter/test_notebook_converter.py]: I removed them in the last commit

**Review by andrzejdackiewicz — APPROVED** [review]
LGTM!

File: streamingcli/jupyter/notebook_converter.py [hunk 2]
```diff
@@ -159,6 +159,9 @@ def _handle_magic_cell(
             if NotebookConverter._skip_statement(sql_statement):
                 return []
             return self._convert_sql_statement_to_python_instructions(sql_statement)
+        if source.startswith("%%flink_execute"):
+            code = "\n".join(source.split("\n")[1:])
+            return [Code(value=code)]
         if source.startswith("%flink_execute_sql_file"):
```
Comment (MarekMaj) [inline streamingcli/jupyter/notebook_converter.py:165]: I don't think this line will be invoked anytime after adding few lines of code above :)

**Pull request merged**